modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/NavigationButtons.js

```diff
@@ -24,18 +24,24 @@ const NavigationButtons = ({moveController, position}) => {
 				top: position.top,
 			}}
 		>
-			<div className="box top" onClick={() => moveController('up')}>
+			<button className="box top" onClick={() => moveController('up')}>
 				<ClayIcon className="icon" symbol="angle-left" />
-			</div>
-			<div className="box right" onClick={() => moveController('right')}>
+			</button>
```

Comment:
I think He didn't used ClayButtonWithIcon to no have `btn-*` classes, just `box top` classes. However, I think @garofalog, you can use ClayButtonWithIcon with displayType: **unstyled** and pass `box top` as className. It will look like:

```jsx
<ClayButtonWithIcon
  className="box top"
  displayType="unstyled"
  onClick={() => moveController('up')}
  symbol="angle-left"
/>
```

Reply:
i tried all the ways you have proposed. But replacing divs with button, ClayButton or ClayButtonWithIcon the component style look different and the margin on :focus keeps showing also in different ways and that's no even way to add an icon to the svg contained in button

Reply:
Hello guys i retried to make the things work with ClayButtons but it takes a lot of work to override the clay style and add my styles properly to the svg.
Could i solve addin a `type="button"` to divs?

Reply:
Sorry, I didn't saw this comment 😅 

I suggested using an anchor element in https://github.com/liferay-frontend/liferay-portal/pull/1242#discussion_r670500090